### ISSUE TITLE
About trainging masks dataset

### ISSUE THREAD

**Issue by haolin512900** [lyndonzheng/Pluralistic-Inpainting#57]
Hi，Thank you for your great project！
     Is your masks training dataset from pconv provided？Is your training set from pconv? If so, how do you divide it?

**Comment by lyndonzheng** [comment]
Hi @haolin512900, in our CVPR19 version, we did not use the free-form mask of pconv. However, we added it to the data processing. Please refer to the 'mask_type==3' in the /dataloader/data_loader.py.

**Comment by haolin512900** [comment]
> Hi @haolin512900, in our CVPR19 version, we did not use the free-form mask of pconv. However, we added it to the data processing. Please refer to the 'mask_type==3' in the /dataloader/data_loader.py.

Hi@lyndonzheng ,So you use "mask_type==1,2" for the training?

**Comment by lyndonzheng** [comment]
Hi @haolin512900, in this CVPR19 version, we only use mask_type=1,2 for the training.

**Comment by haolin512900** [comment]
Thank you for your answer，good luck
